crates/volta-core/src/fs.rs

```diff
@@ -120,14 +120,12 @@ pub fn create_staging_dir() -> Fallible<TempDir> {
 
 /// Create a file symlink. The `dst` path will be a symbolic link pointing to the `src` path.
 pub fn symlink_file<S, D>(src: S, dest: D) -> io::Result<()>
-where
-    S: AsRef<Path>,
-    D: AsRef<Path>,
+    where
+        S: AsRef<Path>,
+        D: AsRef<Path>,
 {
-    #[cfg(windows)]
     return std::os::windows::fs::symlink_file(src, dest);
 
-    #[cfg(unix)]
```

Comment:
I'm not totally sure I follow these changes. If I'm reading the diff correctly (definitely not a guarantee), it looks like this function now unconditionally returns the windows symlink call.

If we're now only calling this on Unix, we can probably remove this function entirely and call `std::os::unix::fs::symlink` directly where we would otherwise call this function. It was originally intended as a helper to simplify dispatching to the correct standard library function based on the OS.

Reply:
I removed the function as advised.